src/routing/wrappers/Layout.svelte

```diff
@@ -14,9 +14,11 @@
   $: isProbeDetailsView = $store.route.section === 'probe';
 
   const defaultTitle = 'GLAM: Glean Aggregated Metrics Explorer';
+  // Use the proper probe name if available (for Glean applications),
+  // falling back to the probeName state if not
   $: title =
     $store.route.section === 'probe'
-      ? `${$store.probeName} | GLAM`
+      ? `${$store.probe.name || $store.probeName} | GLAM`
```

Comment:
I'm wondering if using the new metric name format (e.g. old: `avif.dav1d_decode_error` vs new: `avif_dav1d_decode_error`) will affect the `View SQL Query` feature, since it seems like we're using the old format for our queries? 

What do you think @robhudson, if you're available?